src/core/math.d

```diff
@@ -76,7 +77,19 @@ real sin(real x) @safe pure nothrow;       /* intrinsic */
  * indeterminate.
  */
 version (LDC)
-    alias rndtol = stdc.llroundl;
+{
+    // use a fake-pure C llround[l] (might actually touch C errno)
+    static if (real.sizeof == double.sizeof)
+    {
+        private extern(C) long llround(real x) @safe pure nothrow;
+        alias rndtol = llround;
```

Comment:
I prefer using the C double version in case `real` is equivalent to `double`, because some of these C runtimes don't even provide that symbol (using a macro in the C headers etc.).